translations/locales/en-US/translations.json

```diff
@@ -307,5 +307,137 @@
     "Or": "Or",
     "AlreadyHave": "Already have an account?",
     "Login": "Log In"
+  },
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
What are all these spaces for?

Reply:
Trying to "reserve" spaces in my branches, to deal better with the merging process in JSON file. Nevermind, I'll just get rid of the lines.